test/jest.config.ui.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import config from '../../../src/dev/jest/config';
+
+export default {
+  ...config,
+  roots: ['<rootDir>/plugins/opendistro_security'],
```

Comment:
Please add 

~~~
  moduleNameMapper: {
    '\\.(css|less|sass|scss)$': '<rootDir>/__mocks__/styleMock.js',
    '\\.(gif|ttf|eot|svg)$': '<rootDir>/__mocks__/fileMock.js',
  },
~~~

here, otherwise importing scss and svg in tsx will cause failure in unit test.

Reply:
this is taken care by the import clause `import config from '../../../src/dev/jest/config';` which includes the required config, https://github.com/elastic/kibana/blob/v7.9.1/src/dev/jest/config.js#L62-L64

Reply:
Got it. Just to confirm, in this case we don't need `__mocks__/styleMock.js` and `__mocks__/fileMock.js` in this repo, correct?